src/charts/stacked-area.js

```diff
@@ -1171,6 +1247,21 @@ define(function(require){
             return this;
         };
 
+        /**
+         * Gets or Sets the emptyDataConfig of the chart
+         * @param  {Object} _x emptyDataConfig object to get/set
+         * @return { emptyDataConfig | module} Current config for when chart data is an empty array
```

Comment:
Let´s do Object instead of emptyDataConfig here (other accessors are doing this wrong)